### ISSUE TITLE
Not able to run the app

### ISSUE THREAD

**Issue by mhavelka77** [elovation/elovation#134]
After ~1 hour trying to setup the application I gave up and moved on to another solution.

# problem 1

When following the setup guide on Windows, build fails at 
```RUN SECRET_KEY_BASE=DUMMY ./bin/rails assets:precompile```
very likely due to line ending handling.

# problem 2
When running on wsl, I successfully built the app, however it kept rebooting on the fly.io platform

**Comment by jakelooker** [comment]
What was the other solution that you ended up using?

**Comment by mhavelka77** [comment]
A blackboard 🤣

**Comment by christoshrousis** [comment]
Can you check to see if you're running into the same issue stated in this thread?
https://github.com/elovation/elovation/issues/115

**Comment by mhavelka77** [comment]
Yes, the first problem I described is the same as the ticket you mentioned - Linux vs Windows line ending.
I don't know what the second problem is as I gave up at that point. I didn't save any logs from fly.io ....
